cranelift/codegen/src/machinst/abi_impl.rs

```diff
@@ -310,7 +310,7 @@ pub trait ABIMachineSpec {
         params: &[ir::AbiParam],
         args_or_rets: ArgsOrRets,
         add_ret_area_ptr: bool,
-    ) -> CodegenResult<(Vec<ABIArg>, i64, Option<usize>)>;
+    ) -> CodegenResult<(SmallVec<[ABIArg; 6]>, i64, Option<usize>)>;
```

Comment:
little nit: we can use `ABIArgVec` here too I think...

Reply:
Thanks! missed that one